### PR TITLE
[release/v2.26] Bump KubeVirt CSI Driver Operator (#13833)

### PR DESCRIPTION
**Pull request by moadqassem**
**What this PR does / why we need it**:
This is a manual cherry-pick of  #13833
**Which issue(s) this PR fixes**:

Fixes #

**What type of PR is this?**

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change? Then add your Release Note here**:

```release-note
Bump KubeVirt CSI Driver Operator to support zone-aware topologies
```

**Documentation**:

```documentation
None
```

### CODE REVIEW

**Review by embik — APPROVED** [review]
/approve

**Comment by embik** [comment]
/hold

@moadqassem the verify job is failing.

File: addons/csi/kubevirt/csi-driver-operator.yaml [hunk 1]
```diff
@@ -377,5 +399,17 @@ spec:
...
+      - {{ . }}
+{{- end }}
+     zones:
+{{- range .Zones }}
+      - {{ . }}
+{{- end }}
+     labels:
+{{- range $key, $value := .Labels }}
+       {{ $key }}: {{ $value }}
+{{- end }}
```
Comment (embik) [inline addons/csi/kubevirt/csi-driver-operator.yaml]: This is a little late for me to realise but ... these are all optional fields which default to nil. If you upgrade from 2.26.0, won't this result in a file that looks like this:

```yaml
[...]
bus: scsi
regions:
zones:
labels:
```

Isn't that invalid YAML? 🤔

**Comment by embik** [comment]
/test pre-kubermatic-e2e-kubevirt-ubuntu-1.31

It looks like we never ran this (not on the original PR either), so let's check ...

File: addons/csi/kubevirt/csi-driver-operator.yaml (hunk 1 above)
Comment (moadqassem) [reply addons/csi/kubevirt/csi-driver-operator.yaml]: Well, this actually has been tested and it results something like this.
```yaml
    storageClasses:
    - bus: scsi
      infraStorageClassName: sc-san-standard-hh
      isDefaultClass: false
      labels:
        topology.kubernetes.io/region: central
        topology.kubernetes.io/zone: hh
      regions:
      - central
      volumeBindingMode: WaitForFirstConsumer
      zones:
      - hh
    - bus: scsi
      infraStorageClassName: sc-san-standard-fr
      isDefaultClass: false
      labels:
        topology.kubernetes.io/region: central
        topology.kubernetes.io/zone: fr
      regions:
      - central
      volumeBindingMode: WaitForFirstConsumer
      zones:
      - fr
 ```

**Review by embik — APPROVED** [review]
/approve

**Pull request merged**